ext/pdo_oci/oci_statement.c

```diff
@@ -793,15 +793,225 @@ static int oci_stmt_get_col(pdo_stmt_t *stmt, int colno, char **ptr, size_t *len
 	}
 } /* }}} */
 
+
+static int oci_stmt_col_meta(pdo_stmt_t *stmt, zend_long colno, zval *return_value) /* {{{ */
+{
+	pdo_oci_stmt *S = (pdo_oci_stmt*)stmt->driver_data;
+	pdo_oci_column *C;
+	OCIParam *param = NULL;
+	OraText *colname;
+	OraText * schema;
+	ub2 dtype, data_size;
+	ub2 precis = 0;
+	ub4 namelen, schemalen, typelen, objlen;
+	sb1 scale;
```

Comment:
What do you think about changing the definition of getColumnMeta for Oracle and also adding the scale to the output array?  In Oracle-land, the precision & scale go together.

Reply:
Well, IMHO it's a good idea, Oracle provide the scale so why not add it too.
Maybe with 'oci:scale' or just 'scale'.

Reply:
I'd vote 'scale'.